src/store/sse-context.tsx

```diff
@@ -50,9 +50,18 @@ export const SSEContext = createContext<SSEContextType>({
   setInstallingAppId: () => {},
 });
 
+// for personal development - change backend with .env file
+var backendserver = process.env.REACT_APP_BACKEND;
+if (
+  process.env.REACT_APP_BACKEND == null ||
+  process.env.REACT_APP_BACKEND === ""
+) {
+  backendserver = "http://localhost:8000";
+}
 export const SSE_URL = window.location.hostname.includes("localhost")
-  ? "http://localhost:8000/sse/subscribe"
+  ? backendserver + "/api/sse/subscribe"
   : "/api/sse/subscribe";
+console.log("Running with backend server: " + backendserver);
```

Comment:
nit:
```js
console.info("Running with backend server: ", backendserver);
```

`.log` is usually used for debugging. Some linters complain about it. `.info` makes it clear that this is a wanted output.